CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## [_Unreleased_](https://github.com/freckle/amazonka-mtl/compare/v0.1.0.0...main)
```

Comment:
Does the version number need a bump here too?

Reply:
Yup, will fix.